src/Images/Images.Application/Models/ImageOutputModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BuildingMarket.Images.Application.Models
+{
+    public class ImageOutputModel
+    {
+        public string DisplayUrl { get; set; }
```

Comment:
no attribute? Does that work?

Reply:
Yes, it’s working. The default serialization makes the property what the front-end expects (displayUrl). In the ImageData model, we use the attribute because of the deserialization of the required data from the response.